ebi-migration.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: idr
+title: IDR EBI migration
+---
+<!-- begin marketing header -->
+<header class="marketing-hero">
+    <div class="row homepage text-center">
+        <div class="medium-12 columns">
+            <h1 class="hero-main-header">The Image Data Resource is currently running with reduced functionality</h1>
+        </div>
+    </div>
+</header>
+<!-- end marketing header -->
+
+<div class="row">
+    <div class="medium-12 columns">
+        <p>
+            During March 2020 the EMBL-EBI data centre which hosts the IDR is migrating to a new physical location.
+            From 11th March until roughly the end of the month the following IDR services will be unavailable:
+        </p>
+        <ul stylex="text-align:left;">
+            <li>Virtual Analysis Environment (VAE)</li>
+            <li>FTP submissions</li>
+        </ul>
+        <p>
+            Web and API access to the <a href="https://idr.openmicroscopy.org/">IDR</a> will continue to function, thanks to the provision of a temporary data centre.
```

Comment:
```suggestion
            Web and API access to the <a href="https://idr.openmicroscopy.org/">IDR</a> will continue to function, thanks to the provision of a temporary data center.
```
Are we still doing American spelling?